examples/parallel/distributed_backend_simple.py

```diff
@@ -0,0 +1,44 @@
+"""
+Using distributed for single_machine parallel computing
+========================================================
+
+Realistic usage scenario: combining dask code with joblib code, for
+instance using dask for preprocessing data, and scikit-learn for machine
+learning.
+
+"""
+
+###############################################################################
+# Setup the distributed client
+###############################################################################
+from distributed import Client
+# Typically, to execute on a remote machine, the address of the scheduler
+# would go there
+client = Client()
+
+# This import registers the dask backend for joblib
+import distributed.joblib
+
+###############################################################################
+# Run parallel computation using dask.distributed
+###############################################################################
+
+import time
+import joblib
+
+def long_running_function(i):
+    time.sleep(.1)
+    return i
+
+with joblib.parallel_backend('dask.distributed',
+                             scheduler_host=client.scheduler.address):
```

Comment:
`client.scheduler.address` isn't always available, depending on how the type of cluster (It is available for the `LocalCluster` created automatically for `Client()`, not necessarily for others)

I believe the recommended way would be

```python
address = client.scheduler_info()['address']

with joblib.parallel_backend('dask.distributed',
                             scheduler_host=address): 
```

Reply:
client.scheduler.address *is* always available, in this case `scheduler` is actually the connection object, not the scheduler itself.  In either event though you don't need either.

Also, if relevant, I hope to release dask.distributed within a week.